sdk/storage/storage-blob/src/BlobServiceClient.ts

```diff
@@ -160,15 +164,49 @@ export class BlobServiceClient extends StorageClient {
    * @returns {ContainerClient}
    * @memberof BlobServiceClient
    */
-  public createContainerClient(
-    containerName: string
-  ): ContainerClient {
+  public createContainerClient(containerName: string): ContainerClient {
     return new ContainerClient(
       appendToURLPath(this.url, encodeURIComponent(containerName)),
       this.pipeline
     );
   }
 
+  /**
+   * Create a Blob container.
+   *
+   * @param {string} containerName Name of the container to create.
+   * @param {ContainerCreateOptions} [options] Options to configure Container Create operation.
+   * @returns {Promise<{ containerClient: ContainerClient; response: Models.ContainerCreateResponse }>} Container creation response and the corresponding container client.
+   * @memberof BlobServiceClient
+   */
+  public async createBlobContainer(
```

Comment:
container is for blob service only, sounds little strange for "blobContainer". Why not use `createContainer`?

Reply:
`.NET` has `createBlobContainer` but yes we can use `createContainer`.